packages/pharos/src/components/image-card/pharos-image-card.test.ts

```diff
@@ -810,4 +810,13 @@ describe('pharos-image-card', () => {
     );
     expect(checkboxElement).not.to.be.null;
   });
+
+  it('renders the image preview not available in error state ', async () => {
+    component.error = true;
+    await component.updateComplete;
+
+    const imagePreview = component.renderRoot.querySelector('.unavailable-text');
+    expect(imagePreview).to.exist;
+    expect(imagePreview?.innerHTML).equals('Image preview not available');
+  });
```

Comment:
We typically take a user-first testing approach, so avoid selecting things by their DOM contents when possible. I wonder if we can just check if `component.renderRoot` contains `'Image preview not available'` in some way without knowing about `.unavailable-text`?

Reply:
```
    component.error = true;
    await component.updateComplete;
    expect(component.renderRoot.textContent).to.contain('Image preview not available');
```
would the modifications above be suitable?

Reply:
Yeah I think that is an improvement!